src/components/ReviewList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import { getReviews } from "../api";
+import { ReviewCard } from "./ReviewCard";
+
+export const ReviewList = () => {
+    const [searchParams, setSearchParams] = useSearchParams();
+    const categoryQuery = searchParams.get('category');
+    const [listOfReviews, setListOfReviews] = useState([]);
+
```

Comment:
Can we add some loading functionality to feedback to the user while they wait for the page to load?